src/main/webapp/app/genomic/genomic.component.ts

```diff
@@ -116,4 +123,21 @@ export class GenomicComponent implements OnInit {
     unCheckRadio(key, event) {
         this.genomicInput[key] = this.mainutilService.unCheckRadio(this.genomicInput[key], event.target.value);
     }
+    changeGeneset() {
+        if (this.genomicInput.geneset_id) {
+            this.disableHugoSymbol = true;
+            this.selectedGenesetOption = _.find(this.genesetOptions, { id: this.genomicInput.geneset_id });
+            this.genomicInput.hugo_symbol = this.selectedGenesetOption.genes.join(', ');
+            this.genomicInput.geneset = this.selectedGenesetOption.name;
```

Comment:
If the name is updated, what's the process updating the name? Also if the geneset is updated(add/remove gene), what's the process?

Reply:
When a genomic node is initialized, I'll use the geneset_id to find the corresponding options pulled from OncoKB API (line 66 ~ 68). Even if a geneset name changed or its genes added or removed, the curation platform will always use its id to get latest data.

Reply:
Besides, I'll always use geneset_id to fetch latest `hugo_symbol` from OncoKB API during trial matching. The `hugo_symbol` in trial is only used to display to curators.